src/libs/asyncOpenURL/index.desktop.js

```diff
@@ -0,0 +1,10 @@
+import {Linking} from 'react-native';
```

Comment:
It's been a while so I'm not 100% sure, but does it work to provide an index.website.js for the web implementation and just index.js for all other platforms?

Reply:
@roryabraham It seems to work! Would that be preferred? If so, I will also go ahead and update the docs [here](https://github.com/Expensify/Expensify.cash#platform-specific-file-extensions)